### ISSUE TITLE
Vignette fails to build on Linux

### ISSUE THREAD

**Issue by sjmgarnier** [sjmgarnier/viridis#19]
From CRAN maintainer Kurt Hornik: 

```
* checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: raster
Loading required package: sp

Attaching package: 'raster'

The following object is masked from 'package:colorspace':

    RGB

Loading required package: lattice
Loading required package: latticeExtra
Loading required package: RColorBrewer

Attaching package: 'latticeExtra'

The following object is masked from 'package:ggplot2':

    layer

Quitting from lines 221-248 (intro-to-viridis.Rmd)
Error: processing vignette ‘intro-to-viridis.Rmd’ failed with diagnostics:
there is no package called 'mapproj'
Execution halted
```

@noamross I guess we need to ditch the maps from the vignette. The package was pushed to CRAN, so we need to send an update as soon as possible. What do you think?

**Comment by sjmgarnier** [comment]
Actually, I guess just adding mapproj to the list of suggested packages in the DESCRIPTION should be enough, right?

**Comment by sjmgarnier** [comment]
After talking with CRAN maintainer, it seems that we should add mapproj as a suggested package in the DESCRIPTION file. I will do it, make a minor version bump (0.3.1) because 0.3 was already pushed to CRAN, and then close this issue if everything goes well this time.

**Comment by sjmgarnier** [comment]
Closed by 476f52bcc9cd67327c6d12fb7045413296ecb404.
